project/ianlienfa/README.rst

```diff
@@ -0,0 +1,122 @@
+================================================
+BBGym : The branch and bound framework 
+================================================
+
+Project name: BBGym_ 
+.. BBGym_: <https://github.com/ianlienfa/BBGym>
+
+This is a branch and bound problem solving framework for all kinds of OR problems (especially scheduling problems)
+The goal of this framework is to make it easy to toggle between different search strategies, branching strategies and pruning strategies
+The framework aim to construct environment for Reinforcement Learning the branch and search strategies on branch and bound
+
+Problem to Solve
+================
+
+This framework is aim for two goals:
```

Comment:
"aims for".  Grammar and spelling are not really an issue in this course, but it does not hurt to be careful about it.  I will not nitpick like this in the future.